README.md

```diff
@@ -86,6 +86,9 @@ If you'd like to unsubscribe, send an email with the word *unsubscribe* to cfs-c
 
 You can email the cFS Product Team at cfs-program@lists.nasa.gov to explore partnerships and other arrangements for in-depth support.
 
+### Discussions 
```

Comment:
Can we move this before the mailing list? I want people to use that more. Also, we might want to remove the line about submitting issues from the  text immediately `## Getting Help`